{"name": "test_valid_login[credentials0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = <bound method LoginPage.is_dashboard_text_visible of <src.pages.login.login_page.LoginPage object at 0x000001C32C313490>>()\n +    where <bound method LoginPage.is_dashboard_text_visible of <src.pages.login.login_page.LoginPage object at 0x000001C32C313490>> = <src.pages.login.login_page.LoginPage object at 0x000001C32C313490>.is_dashboard_text_visible", "trace": "driver_manager = <src.utils.driver_manager.DriverManager object at 0x000001C32C310510>, credentials = {'password': 'Password123', 'username': 'student'}\n\n>   ???\nE   assert False\nE    +  where False = <bound method LoginPage.is_dashboard_text_visible of <src.pages.login.login_page.LoginPage object at 0x000001C32C313490>>()\nE    +    where <bound method LoginPage.is_dashboard_text_visible of <src.pages.login.login_page.LoginPage object at 0x000001C32C313490>> = <src.pages.login.login_page.LoginPage object at 0x000001C32C313490>.is_dashboard_text_visible\n\nd:\\Selenium Project\\web_automation_selenium - Copy\\src\\tests\\login\\test_login.py:23: AssertionError"}, "parameters": [{"name": "credentials", "value": "{'username': 'student', 'password': 'Password123'}"}], "start": 1730959371970, "stop": 1730959375451, "uuid": "23b6d0e9-eb5b-4ab9-8b05-a07c93815408", "historyId": "9508d0765b0b1f8bcccda8a6b0c295ef", "testCaseId": "2e11a866fb8562c8d371512e0099dafd", "fullName": "src.tests.login.test_login#test_valid_login", "labels": [{"name": "parentSuite", "value": "src.tests.login"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DESKTOP-1ML9BAE"}, {"name": "thread", "value": "1336-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.login.test_login"}]}